lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -1296,42 +1292,38 @@
 
   // CASE: is denied
   &.is-denied {
+    border-top: 3px solid $red !important;
```

Comment:
@tasawernawaz so without adding the ```!important``` does it not work? Wherever possible we should avoid using important if we can.